service/src/integrationTest/kotlin/fi/espoo/evaka/messaging/MessageNotificationEmailServiceIntegrationTest.kt

```diff
@@ -106,6 +106,8 @@ class MessageNotificationEmailServiceIntegrationTest :
         }
     }
 
+    val SPREAD_MESSAGE_NOTIFICATION_SECONDS: Long = 60 * 60 * 24
```

Comment:
Instead of duplicating the constant here, where it can get out of sync and lead to integration test flakiness, you could have the value only in the MessageService companion object, and access it in both the test and real implementation as `MessageService.SPREAD_MESSAGE_NOTIFICATION_SECONDS`